articles/logic-apps/quickstart-create-first-logic-app-workflow.md

```diff
@@ -56,17 +56,17 @@ Sign in to the [Azure portal](https://portal.azure.com) with your Azure account
 
    Or, you can find and select your logic app by typing the name in the search box.
 
-   The Logic Apps Designer opens and shows a page with an introduction video and commonly used triggers. Under **Templates**, select **Blank Logic App**.
+   The logic apps Designer opens and shows a page with an introduction video and commonly used triggers. Under **Templates**, select **Blank logic app**.
 
    ![Select blank template for logic app](./media/quickstart-create-first-logic-app-workflow/choose-logic-app-template.png)
 
-Next, add a [trigger](../logic-apps/logic-apps-overview.md#logic-app-concepts) that fires when a new RSS feed item appears. Every logic app must start with a trigger, which fires when a specific event happens or when a specific condition is met. Each time the trigger fires, the Logic Apps engine creates a logic app instance that starts and runs your workflow.
+Next, add a [trigger](../logic-apps/logic-apps-overview.md#logic-app-concepts) that fires when a new RSS feed item appears. Every logic app must start with a trigger, which fires when a specific event happens or when a specific condition is met. Each time the trigger fires, the logic apps engine creates a logic app instance that starts and runs your workflow.
 
 <a name="add-rss-trigger"></a>
 
 ## Check RSS feed with a trigger
 
-1. In Logic App Designer, under the search box, select **All**.
+1. In logic app Designer, under the search box, select **All**.
```

Comment:
"Logic App Designer" is the user interface element and designer name, which is capitalized. Please change back to "Logic App Designer", thanks!